opencog/util/exceptions.cc

```diff
@@ -100,11 +100,9 @@ StandardException::StandardException(const StandardException& ex)
 
 StandardException& StandardException::operator=(const StandardException& ex)
 {
-    message = NULL;
     if (ex.message)
     {
-        message = new char[strlen(ex.message) + 1];
-        strcpy(message, ex.message);
+        set_message(ex.message);
```

Comment:
Cool. It seems to me, you can apply the same simplification in the constructor, if you initialize message to `NULL` (or rather `nullptr`).

Reply:
Yeah, I've thought about doing the same thing in the constructor but then decided to not touch it for some reason.